elephant/datasets.py

```diff
@@ -129,13 +129,13 @@ def download_datasets(repo_path, filepath=None, checksum=None,
 
         >>> import os
         >>> from elephant.datasets import download_datasets
-        >>> os.environ["ELEPHANT_DATA_URL"] = "https://web.gin.g-node.org/INM-6/elephant-data/raw/multitaper" # noqa
+        >>> os.environ["ELEPHANT_DATA_URL"] = "https://web.gin.g-node.org/NeuralEnsemble/elephant-data/raw/multitaper" # noqa
```

Comment:
The doctests fail at this line, since there is no "multitaper" branch on https://web.gin.g-node.org/NeuralEnsemble/elephant-data/ .

**Possible solutions**
- skip this doctest by adding `# doctest: +SKIP`
- create a "multitaper" branch on _NeuralEnsemble/elephant-data/_
- change link to https://web.gin.g-node.org/NeuralEnsemble/elephant-data/raw/master

Reply:
We decided to skip this doctest